app.js

```diff
@@ -66,9 +68,10 @@ if (confirmQuiz){
       }
     }
   }
+
 //calling the function
   questionSix();
```

Comment:
It'll be a lot easier for you if you group all your function calls at the bottom of the file so that it's easy to find stuff.